XRTK-Core/Packages/com.xrtk.core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -95,7 +95,7 @@ public class ControllerPopupWindow : EditorWindow
         private BaseMixedRealityControllerMappingProfile mappingProfile;
 
         private bool IsCustomController => currentControllerType == SupportedControllerType.GenericOpenVR ||
-                                           currentControllerType == SupportedControllerType.GenericUnity;
+                                           currentControllerType == SupportedControllerType.GenericUnity || currentControllerType == SupportedControllerType.Hand;
```

Comment:
Do we need to even show the controller mapping window for hands?

I'd assume everything that was near field interactions would just be driven by physics and the far field interactions would be defined in the gesture recorder or something.